<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="com.gapi.testCases.GetCrossConnectDetails">
        <methods>
          <include name="verifyGetCrossConnectDetailsByInvalidLimitValue" invocation-numbers="0"/>
          <include name="verifyGetCrossConnectDetailsWithNonIntergerOffset" invocation-numbers="0"/>
          <include name="verifyGetCrossConnectDetailsWithNonIntegerLimitValues" invocation-numbers="0"/>
          <include name="verifyGetCrossConnectDetailsWithNegativeLimitValues" invocation-numbers="0"/>
          <include name="generateAuthorizationToken"/>
          <include name="AfterSuite"/>
          <include name="verifyGetCrossConnectDetailsWithNegativeOffset" invocation-numbers="0"/>
          <include name="BeforeSuite"/>
          <include name="verifyCrossConnectDetailsUsingDifferentFilters" invocation-numbers="6"/>
          <include name="teardown"/>
        </methods>
      </class> <!-- com.gapi.testCases.GetCrossConnectDetails -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
